discord/user.py

```diff
@@ -422,20 +422,24 @@ def _update(self, data: UserPayload) -> None:
 
     # TODO: Username might not be able to edit anymore.
     async def edit(
-        self, *, username: str = MISSING, avatar: bytes = MISSING
+        self,
+        *,
+        username: str = MISSING,
+        avatar: bytes = MISSING,
+        banner: bytes = MISSING,
     ) -> ClientUser:
         """|coro|
 
         Edits the current profile of the client.
 
         .. note::
 
-            To upload an avatar, a :term:`py:bytes-like object` must be passed in that
+            To upload an avatar or banner, a :term:`py:bytes-like object` must be passed in that
             represents the image being uploaded. If this is done through a file
             then the file must be opened via ``open('some_filename', 'rb')`` and
             the :term:`py:bytes-like object` is given through the use of ``fp.read()``.
 
-            The only image formats supported for uploading is JPEG and PNG.
+            The only image formats supported for uploading is JPEG and PNG and GIF.
 
         .. versionchanged:: 2.0
             The edit is no longer in-place, instead the newly edited client user is returned.
```

Comment:
Add a version changed
```suggestion
        .. versionchanged:: 2.0
            The edit is no longer in-place, instead the newly edited client user is returned.
            
        .. versionchanges:: 2.6
            The ``banner`` keyword-only parameter was added.
```